src/sparseml/transformers/utils/export.py

```diff
@@ -0,0 +1,252 @@
+    the given model path weights, config, and tokenizer
+
+    :param task: task to create the model for. i.e. mlm, qa, glue, ner
+    :param model_path: path to directory where model files, tokenizers,
+        and configs are saved. ONNX export will also be written here
+    :param sequence_length: model sequence length to use for export
+    :param convert_qat: set True to convert a QAT model to fully quantized
+        ONNX model. Default is True
+    :param finetuning_task: optional string finetuning task for text classification
+        and token classification exports
+    :param onnx_file_name: name to save the exported ONNX file as. Default
+        is model.onnx. Note that when loading a model directory to a deepsparse
+        pipeline, it will look only for 'model.onnx'
+    :return: path to the exported ONNX file
+    """
+    if task.lower() not in _TASK_TO_CONSTRUCTOR:
+        raise ValueError(
+            f"task {task} unsupported for export_transformer_to_onnx. Supported "
+            f"tasks include {list(_TASK_TO_CONSTRUCTOR.keys())}"
+        )
+    auto_model_constructor = _TASK_TO_CONSTRUCTOR[task.lower()]
+
+    if not os.path.isdir(model_path):
+        raise ValueError(
+            "model_path must be a directory that contains the trained transformer "
+            f"files. {model_path} is not a directory"
+        )
+
+    # load config and tokenizer
+    config_args = {"finetuning_task": finetuning_task} if finetuning_task else {}
+    config = AutoConfig.from_pretrained(model_path, **config_args)
+    tokenizer = AutoTokenizer.from_pretrained(
+        model_path, model_max_length=sequence_length
+    )
+
+    # load model
+    model = auto_model_constructor.from_pretrained(
+        model_path,
+        from_tf=False,
+        config=config,
+    )
+
+    # apply recipe if exists before loading model weights
+    recipe_path = os.path.join(model_path, RECIPE_NAME)
+    if os.path.isfile(recipe_path):
+        ScheduledModifierManager.from_yaml(recipe_path).apply(model)
+    else:
+        _LOGGER.warning(f"recipe not found under {recipe_path}")
+
+    # load weights
+    state_dict = torch.load(os.path.join(model_path, WEIGHTS_NAME))
+    model.load_state_dict(state_dict)
+
+    # create fake model input
+    inputs = tokenizer(
+        "", return_tensors="pt", padding=PaddingStrategy.MAX_LENGTH.value
+    ).data  # Dict[Tensor]
+
+    # run export
+    onnx_file_path = os.path.join(model_path, onnx_file_name)
+    export_onnx(
+        model,
+        inputs,
+        onnx_file_path,
+        convert_qat=convert_qat,
+    )
+
+    return onnx_file_path
+
+
+def parse_args() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+        description="Export a trained transformers model to an ONNX file"
+    )
+
+    parser.add_argument(
+        "--task",
+        type=str,
+        required=True,
+        help="task to create the model for. i.e. mlm, qa, glue, ner",
```

Comment:
nit: Use title case for first word, `Task`. Or make every help string start with lowercase